ichnaea/data/tests/test_export.py

```diff
@@ -348,6 +366,18 @@ def test_age(self):
             "wifi": [{"age": -1500}, {"age": 500}],
         }
 
+    def test_null_position(self):
+        assert self.transform(
```

Comment:
The fix was in ``InternalTransform`` helper class. This is as close as the code gets to a unit test to confirm the fix.